lansa-vmss-windows-autoscale-existing-db/azuredeploy.json

```diff
@@ -480,9 +458,9 @@
                               {
                                  "name": "[concat( 'db',variables('ipConfigName'))]",
                                  "properties": {
-                                    "subnet": {
-                                       "id": "[concat('/subscriptions/', subscription().subscriptionId,'/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Network/virtualNetworks/', variables('virtualNetworkName'), '/subnets/', variables('subnetName'))]"
-                                    },
+                                  "subnet": {
+                                    "id": "[concat('/subscriptions/', subscription().subscriptionId,'/resourceGroups/NARS-ISOLATED-NE-NETWORK/providers/Microsoft.Network/virtualNetworks/', variables('virtualNetworkName'), '/subnets/', variables('subnetName'))]"
```

Comment:
I don't think this is going to work at run-time - there's no guarantee the subscription will have a resourceGroup with this name...